src/app/modules/submission/encrypt-submission/encrypt-submission.middleware.ts

```diff
@@ -187,14 +187,17 @@ export const validateSubmission = async (
     .map(() => next())
     .mapErr((error) => {
       logger.error({
-        message: 'Error processing responses',
+        message:
+          'Error processing responses, but proceeding with submission as submission have been validated client-side',
         meta: logMeta,
         error,
       })
-      const { statusCode, errorMessage } = mapRouteError(error)
-      return res.status(statusCode).json({
-        message: errorMessage,
-      })
+      // TODO(FRM-1318): Uncomment to harden submission validation after validation has similar error rates as email mode forms.
```

Comment:
Curious as to why we're commenting out instead of using growthbook? Is it because we haven't integrated with growthbook on the BE side of things?

Reply:
Ah, yes because we don't have growthbook on BE yet. And I'm hoping storage submission validation will be settled pretty soon...
But I presume I can hide this with the BE feature flag, if that makes more sense 🤔

Reply:
Done in [5b85e4d](https://github.com/opengovsg/FormSG/pull/6696/commits/5b85e4dc5561da26f51a9f95ce56b53cbd822f1d) and [300d44a](https://github.com/opengovsg/FormSG/pull/6696/commits/300d44aaa08fdebbbac79ed86cfb5cfc44fdc62e)!